spdx-utils/src/main/kotlin/SpdxConstants.kt

```diff
@@ -35,6 +35,11 @@ object SpdxConstants {
      */
     const val NOASSERTION = "NOASSERTION"
 
+    /**
+     * The URL that points to list of SPDX licenses on https://spdx.org.
```

Comment:
Sorry, one more nit: I believe it does not make much sense to repeat (parts of) the URL as part of the docs, so I'd just drop " on https://spdx.org" here.